schema.development.kf

```diff
@@ -90,11 +90,32 @@ table external_access_grants {
     tx_hash text,
     block_hash text,
     height int,
-    status text,
+    // data_id in contract can be any data, but idOS expects uuid of a credential, so we need a status field to mark validity of the access grant
+    status text, // valid, invalid_by_reason, ...
```

Comment:
I was expecting this on `internal_access_grants`, definitely not on `credentials`.

Maybe we need to ping-pong over this?

Reply:
Misread. Ignore this!